.dockerignore

```diff
@@ -0,0 +1 @@
+.gitignore
```

Comment:
This just adds the `.gitignore` to Docker's ignore list, I assume you meant to do a symlink to `.gitignore`?

Reply:
That would be ideal. though, would it be portable to Windows / NTFS?

Reply:
Interesting, git has its own system of symbolic links. However I don't believe this is required in the first place. I'll remove the file from the PR.